Makefile

```diff
@@ -334,9 +330,6 @@ e2e-gh-token: build-scorecard check-env | $(GINKGO)
 	# Run e2e tests. GITHUB_AUTH_TOKEN set to secrets.GITHUB_TOKEN must be used to run this.
 	TOKEN_TYPE="GITHUB_TOKEN" $(GINKGO) --race -p -v -cover -coverprofile=e2e-coverage.out --keep-separate-coverprofiles ./...
 
-e2e-attestor: ## Runs e2e tests for scorecard-attestor
```

Comment:
@naveensrinivasan Removing this target will cause the integration test to fail since the `integration.yml` file in `main` still calls `make e2e-attestor`. 

Do you think it's better to leave this as an empty target (and remove later) so the job passes? Or would you merge to main with a failing integration test since the new `integration.yml` wouldn't call this in the future? The integration check is still marked as not required, so it's an option.

Reply:
I think we can remove it later. Thanks

Reply:
Going to leave this unresolved until the target is added back.

Reply:
Added it back!